dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/DataQueryParams.java

```diff
@@ -1669,6 +1669,42 @@ public List<DataElement> getSkipTotalDataElements()
             .collect( Collectors.toList() );
     }
 
+    /**
```

Comment:
Moving these methods up to `DataQueryParams` where they belong and can be more easily reused.